{"name": "整体增删改查用户", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'", "trace": "self = <WechatAPI.testcase.test_work.TestWework object at 0x000001FBD74599E8>\nuserid = 'userid7', name = '名字7', mobile = '13800000007'\n\n    @allure.story(\"通讯录管理\")\n    @allure.title(\"整体增删改查用户\")\n    @pytest.mark.parametrize(\"userid,name,mobile\",test_data())\n    def test_wechat(self,userid,name,mobile):\n    \n        # userid=\"zhangsan90\"\n        # name=\"张三思0\"\n        # mobile = \"13290900011\"\n        try:\n>           assert \"created\"==Wework().test_creat_member(userid,name,mobile)[\"errmsg\"]\n\nWechatAPI\\testcase\\test_work.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <WechatAPI.api.we_work.Wework object at 0x000001FBD7459048>\n\n    def __init__(self):\n        self.token = Util().token() #不要忘了Util的()\n        self.params[\"token\"]=self.token #要记得替换token的变量\n    \n>       with open(\"../api/wework.yaml\",encoding=\"utf-8\")as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'\n\nWechatAPI\\api\\we_work.py:27: FileNotFoundError"}, "parameters": [{"name": "userid", "value": "'userid7'"}, {"name": "name", "value": "'名字7'"}, {"name": "mobile", "value": "'13800000007'"}], "start": 1647478472565, "stop": 1647478472930, "uuid": "8446e274-c0f3-4dc7-a61e-e9a9cf33e3d3", "historyId": "77897a31140c5313177940d5b8cb1868", "testCaseId": "6fbae5519000b9212e1162caf7211d2e", "fullName": "WechatAPI.testcase.test_work.TestWework#test_wechat", "labels": [{"name": "story", "value": "通讯录管理"}, {"name": "parentSuite", "value": "WechatAPI.testcase"}, {"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestWework"}, {"name": "host", "value": "ZXJ"}, {"name": "thread", "value": "59216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "WechatAPI.testcase.test_work"}]}